README.md

```diff
@@ -50,12 +52,18 @@ The `quantumarticle` class tries to be **maximally compatible** with existing do
 
 In addition, this document class come bundled with two new extras (currently in beta phase):
 
-1. The quantum-plots.ipynb Jupyter notebook and matplotlib style for effortlessly producing professional looking plots in python.
+1. The quantum-lyx-template.lyx LxY layout, which allow you to generate the LaTeX source of your quantumarticle manuscript with the LyX document processor.
 
-2. The quantum-lyx-template.lyx LxY layout, which allow you to generate the LaTeX source of your quantumarticle manuscript with the LyX document processor.
+2. The quantum.bst bibliography style for BibTeX, see the corresponding demo for usage details.
 
 ## Changelog
 
+### New in v6.0
+
+* introduced the Quantum bibstyle `quantum.bst` together with the demo file `quantum_bst_demo`
```

Comment:
Can we please rename that to quantum-bibliographystyle-demo.tex

Reply:
Done.